### ISSUE TITLE
Corruption when creating a pivot table where some cells are floats

### ISSUE THREAD

**Issue by ses4j** [EPPlusSoftware/EPPlus#440]
(This is similar to the other, fixed report in #410, so I copied some of what I wrote there.  Only the test file is different, with a 1.5 instead of the blank cell.)

Using EPPlus 5.7.1, net5.0.

I'm trying to create a Pivot Table, but my source data has some ints and some floats, and that seems to corrupt the pivotCache somehow. (If I have all ints and blank cells, it works fine.) When I try to open the output file, it says:

> We found a problem with some content in 'test-out.xlsx'. Do you want us to try to recover as much as we can? If you trust the source of this workbook, click Yes.

After clicking Yes, it says:

> Repaired Records: PivotTable report from /xl/pivotCache/pivotCacheDefinition1.xml part (PivotTable cache)

And then it works fine.

Input file:
[test-in.xlsx](https://github.com/EPPlusSoftware/EPPlus/files/6850048/test-in.xlsx)
![image](https://user-images.githubusercontent.com/435529/126365995-a13b843d-7072-4149-b4c3-dc93c84f5129.png)

Test Program.cs:

```
using System;
using System.IO;
using System.CommandLine;
using OfficeOpenXml;
using OfficeOpenXml.Table;
using OfficeOpenXml.Table.PivotTable;

namespace addpivot
{
    public class Class1
    {
        static void Main()
        {
            ExcelPackage.LicenseContext = LicenseContext.NonCommercial;
            using (var package = new ExcelPackage(new FileInfo("test-in.xlsx")))
            {
                var wb = package.Workbook;
                var worksheet = wb.Worksheets.Add("Pivot Tables");
                var table = wb.Worksheets[0].Tables["Table1"];
                ExcelPivotTable pt = worksheet.PivotTables.Add(worksheet.Cells["A1"], table, "PT1");
                pt.RowFields.Add(pt.Fields["ColC"]);
                pt.DataFields.Add(pt.Fields["ColB"]);
                package.SaveAs(new FileInfo("test-out.xlsx"));
            }
        }
    }
}
```

**Comment by JanKallman** [comment]
It seems that when having mixed ints and floats Excel does not want the containsInteger attribute set. Only when it's integers only.

**Comment by ses4j** [comment]
@JanKallman Thanks!  I cannot imagine what it's like to try to reinvent the xlsx format, it must be like walking through the mind of a madman.  ;)

**Comment by JanKallman** [comment]
Fixed in 5.7.1
